bigquery_etl/dryrun.py

```diff
@@ -38,6 +38,19 @@
     from backports.cached_property import cached_property  # type: ignore
 
 
+QUERY_PARAMETER_TYPE_VALUES = {
+    "DATE": "2019-01-01",
+    "DATETIME": "2019-01-01 00:00:00",
+    "TIMESTAMP": "2019-01-01 00:00:00",
+    "STRING": "foo",
+    "BOOL": True,
+    "FLOAT64": 1,
+    "INT64": 1,
```

Comment:
I didn't try every possible alias but `INTEGER` and `FLOAT` (but not `INT`) are accepted as types as well, e.g. `bq query --parameter "val:INTEGER:10" --nouse_legacy_sql "SELECT @val"`.  I would probably think to use those before the 64 versions
```suggestion
    "FLOAT": 1,
    "INT64": 1,
    "INTEGER": 1,
```